ext/package.json

```diff
@@ -12,12 +12,14 @@
         "babel-preset-env": "^1.6.1",
         "babel-preset-react": "^6.24.1",
         "copy-webpack-plugin": "^4.3.1",
+        "minimist": "^1.2.0",
         "react": "^16.2.0",
         "react-dom": "^16.2.0",
         "web-ext": "^2.7.0",
         "webpack": "^3.10.0"
     },
     "dependencies": {
+        "string-replace-loader": "^2.1.1",
```

Comment:
Whoops, was messing around with this to do the mirrorcast APPI ID, but went with Webpack's DefinePlugin plugin (already in used) in the end. Will submit a force push shortly to blast it away.